jinja2/filters.py

```diff
@@ -842,18 +842,18 @@ def do_groupby(environment, value, attribute):
         {% endfor %}
         </ul>
 
-    Additionally it's possible to use tuple unpacking for the grouper and
-    list:
+    Additionally it's possible to use tuple unpacking for the `gender` and
+    `list`:
 
     .. sourcecode:: html+jinja
 
         <ul>
-        {% for grouper, list in persons|groupby('gender') %}
+        {% for gender, list in persons|groupby('gender') %}
             ...
         {% endfor %}
         </ul>
 
-    As you can see the item we're grouping by is stored in the `grouper`
+    As you can see the item we're grouping by is stored in the `gender`
```

Comment:
Use double backticks to denote code in RST:

```rst
the ``gender`` attribute
```

This applies to all the places you used that syntax.